src/Samples/SarifToCsv/SarifCsvColumnWriters.cs

```diff
@@ -114,5 +117,20 @@ public static Action<WriteContext> GetWriter(string columnName)
 
             return writers;
         }
+
+        private static void WriteProperties(WriteContext c)
+        {
+            StringBuilder result = new StringBuilder();
+
+            foreach (string propertyName in c?.Result?.PropertyNames ?? Enumerable.Empty<string>())
+            {
+                if (result.Length > 0) { result.Append("; "); }
+                result.Append(propertyName);
+                result.Append(": ");
```

Comment:
>:  [](start = 31, length = 2)

Just curious why you chose `name: value` over `name=value`. Was it a coin toss or is there a reason `:` is better?

Reply:
I'm reviewing these as a human and find them easier to "parse" visually when there's a space before each distinct property name and value.